ui/src/dashboard/menus/MenuForm.tsx

```diff
@@ -46,15 +46,21 @@ export const MenuForm = ({ onSubmit, menu = null }) => {
       >
         {({ isSubmitting, values }) => (
           <>
-            <div className="card card--full-width card--lightestGray">
-              <Form>
-                <Field isFormik name="title" label="Title" />
-                <Field isFormik as="textarea" label="Description" name="description" />
-                <Field isFormik as="textarea" label="Disclaimer" name="disclaimer" />
-                <Button type="submit" disabled={isSubmitting}>
-                  {menu ? 'Update' : 'Create'} Menu
-                </Button>
-              </Form>
+            <div className="menu-info">avatar img? menu name published? edit, save, preview</div>
+            <div className="card card--full-width card--lightestGray card--update grid">
+              <div className="headerImg-container">
+                <div className="headerImg">header img</div>
+              </div>
+              <div>
+                <Form>
+                  <Field isFormik name="title" label="Title" />
+                  <Field isFormik as="textarea" label="Description" name="description" />
+                  <Field isFormik as="textarea" label="Disclaimer" name="disclaimer" />
+                  <Button type="submit" disabled={isSubmitting}>
+                    {menu ? 'Update' : 'Create'} Menu
+                  </Button>
+                </Form>
+              </div>
```

Comment:
This and the card css code should be in a separate PR than the footer stuff. If for some reason one of these 2 things needs rolledback for example, we couldnt ever cherry pick a specific issue, we would have to remove both features. Just a good habbit to remember and isolate PRs and not double dip when possible :)